lang/locales.js

```diff
@@ -5,7 +5,14 @@ const LOCALES = {
 	de: require('./de.json'),
 };
 
+/**
+ * Returns a locale entry (translated text)
+ * @param {string} text The key to look for in the translation table
+ * @param {string} locale The language code to use. Default is en (english)
+ */
 module.exports.__ = (text, locale) => {
+	if (typeof text != 'string') return text;
```

Comment:
```suggestion
	if (typeof text !== 'string') return text;
```

Reply:
Thanks for the suggestion. I'm still relatively new to JavaScript, coming from FoxPro and C# .Net...

Reply:
No worry, I'm not a programmer myself. JavaScript is just a hobby for me!

Most of the cases, you want to use `===` or `!==`.
The only exception I do, it's when I want to check for a **null** and an **undefined** value: a single `== null` is valid for both.